src/elements/Menu/Menu.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react"
+import styled from "styled-components"
+
+import { color } from "../../helpers/color"
+import { BorderBox } from "../BorderBox"
+import { Box, BoxProps } from "../Box"
+import { Flex } from "../Flex"
+import { Link } from "../Link"
+import { Separator } from "../Separator"
+import { Spacer } from "../Spacer"
+import { Sans } from "../Typography"
+
+interface MenuProps {
+  title?: string
+  children?: React.ReactNode
+}
+
+export const Menu: React.FC<MenuProps> = ({ title, children, ...props }) => {
+  return (
+    <MenuContainer width={230} m="2px" {...props}>
+      <BorderBox p={0} pb={1}>
+        <Flex flexDirection="column" width="100%">
+          {title && (
+            <Box px={2} pt={2} pb={1}>
+              <Sans size="3" weight="medium">
+                {title}
+              </Sans>
+              <Spacer py={0.5} />
+              <Separator />
+            </Box>
+          )}
+
+          <Box pt={title ? 0 : 1}>{children}</Box>
+        </Flex>
+      </BorderBox>
+    </MenuContainer>
+  )
+}
+
+const MenuContainer = styled(Box)`
+  background-color: white;
+  box-shadow: 2px 2px 4px 2px ${color("black5")};
+`
+
+// Menu Item
+
+interface MenuItemProps extends BoxProps {
+  children: React.ReactNode
+  href: string
+  color?: string // TODO:  Look into type conflict with styled-system
+}
+
+export const MenuItem: React.FC<MenuItemProps> = ({
+  children,
+  href,
+  ...props
+}) => {
+  return (
+    <MenuItemContainer onClick={() => window.location.assign(href)} {...props}>
```

Comment:
Capture the outer container click while leaving an `href` link below for seo, screen-readers, etc.

Reply:
I don't follow - why do we need a click handler here?

Reply:
This keeps things a little visually flexible and easily clickable since there are some constraints around nesting block level content within an `a` tag. With a click handler on the outer container I can capture the entire area while still being able to style using common palette idioms.

Reply:
Have we tested this will work with SEO/screen-readers? I've always operated under the rule "if the user clicks on it and it takes them somewhere, use a link", and capturing the click on a div makes me slightly uncomfortable. I don't have any data or resources to point to that say "don't do it", but I'd want to be very sure that it was safe.

Reply:
Ohhh, this is about click target, right? Without that `onClick`, there would be space inside the `MenuItemContainer` where the user wouldn't be able to click. I guess that happens because of the padding on the `Box`, right? Any reason to not promote `Link` up and give it the padding? Are these the idioms you were referring to?

Reply:
I don't have anything offhand, but I can update the container to be an `a` tag. This does seem a bit funny.